enclave-tls/src/attesters/sgx-ecdsa/collect_evidence.c

```diff
@@ -74,28 +99,27 @@ enclave_attester_err_t sgx_ecdsa_collect_evidence(enclave_attester_ctx_t *ctx,
 		close(sgx_fd);
 		return -ENCLAVE_ATTESTER_ERR_INVALID;
 	}
-#else
-	sgx_ecdsa_ctx_t *ecdsa_ctx = (sgx_ecdsa_ctx_t *)ctx->attester_private;
+#elif defined(SGX)
```

Comment:
This statement makes people confusing because we will ask "what happens if using non-sgx/non-occlum build".